sofa-boot-core/src/main/java/com/alipay/sofa/rpc/boot/config/SofaBootRpcProperties.java

```diff
@@ -118,6 +118,8 @@
 
     private String              consumerRepeatedReferenceLimit;
 
+    private String              enableHystrix;
```

Comment:
Does `enableHystrix ` means the key is `com.alipay.sofa.rpc.enable.hystrix` ?

I think `com.alipay.sofa.rpc.hystrix.enable` is better.

Reply:
It should be com.alipay.sofa.rpc.enable-hystrix=true